src/panels/NightSkyPanel/tabs/TimeTab.tsx

```diff
@@ -0,0 +1,188 @@
+import { useEffect, useMemo, useState } from 'react';
+import { Alert, Button, Divider, Group, Stack, Text, Title } from '@mantine/core';
+import * as GeoTZ from 'browser-geo-tz';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { LoadingBlocks } from '@/components/LoadingBlocks/LoadingBlocks';
+import { useSubscribeToTime } from '@/hooks/topicSubscriptions';
+import { useAppSelector } from '@/redux/hooks';
+import { isDateValid } from '@/redux/time/util';
+
+// @TODO (2025-05-19, emmbr) This component needs logic for checking if the used actions
+// exist. However, for this we need to be able to access the actions state using the
+// identifier of the action, so loaving for now
+
+export function TimeTab() {
+  const [lastLat, setLastLat] = useState<number>(0);
+  const [lastLong, setLastong] = useState<number>(0);
+  const [localArea, setLocalArea] = useState<string>('UTC');
+
+  const { latitude: currentLat, longitude: currentLong } = useAppSelector(
+    (state) => state.camera
+  );
+
+  const luaApi = useOpenSpaceApi();
+  const timeCapped = useSubscribeToTime();
+
+  const date = useMemo(() => new Date(timeCapped ?? ''), [timeCapped]);
```

Comment:
```suggestion
  const date = new Date(timeCapped ?? '');
```
Unless I'm missing something, this `useMemo` isn't achieving much since `timeCapped` is only updated every second.